buildlib/pr/main.yml

```diff
@@ -135,8 +198,9 @@ stages:
     - template: tests.yml
       parameters:
         name: gpu
-        demands: ucx_gpu -equals yes
+        demands: ucx_gpu_test -equals yes
         test_perf: 1
+        container: centos7_cuda11
```

Comment:
so we run test_jenkins.sh only on centos7_cuda11, but test_memhooks on all matrix?

Reply:
for all cuda official docker images